wrappers/nodejs/index.js

```diff
@@ -4144,6 +4150,11 @@ const format = {
    * <br>Equivalent to its lowercase counterpart.
    * @type {Integer}
    */
+   FORMAT_Y411: RS2.RS2_FORMAT_Y411,
```

Comment:
Duplicate, remove

Reply:
All the formats are duplicate one for string and one for their enum value